demos/python_demos/llm_python_backend/README.md

```diff
@@ -0,0 +1,44 @@
+## Download model
+
+We are going to use [red-pajama-3b-chat](https://huggingface.co/togethercomputer/RedPajama-INCITE-Chat-3B-v1) model in this scenario.  
+Download the model using `download_model.py` script:
+
+```bash
+pip install -r requirements.txt
+python3 download_model.py
+```
+
+The model will appear in `./model` directory.
+
+## Run Model Server
+
+Mount the `./model` directory with the model.  
+Mount the `./servable` which contains:
+- `model.py` and `config.py` - python scripts which are required for execution and uses optimum-intel framework
```

Comment:
Hugging Face* 
and I'd say utilities instead of framework (because what we mean here is transfromers, distutils etc.). HF feels like something bigger than framework.